custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -272,13 +294,12 @@ def _validate_chat_response(response):
             response_iter = iter(response)
             first_chunk = next(response_iter)
 
-            if getattr(first_chunk, "object", None) == "chat.completion.chunk":
+            if type(first_chunk).__name__ == "ChatCompletionChunk":
```

Comment:
Why not `isinstance` ? 
In order not to import ChatCompletionChunk?

Reply:
Yes. We would need to depend on openai python package to have that.